hostinit/setup.go

```diff
@@ -198,6 +199,7 @@ func (init *HostInit) setupReadyHosts(ctx context.Context) error {
 	for i := 0; i < numThreads; i++ {
 		wg.Add(1)
 		go func() {
+			defer recovery.LogStackTraceAndExit("setupReadyHosts")
```

Comment:
I think continuing is probably right here? both send slack alerts and exiting is probably only right if we want to stop all threads as a result of this error. but that it'll restart because of the way systemd will restart things.